models/linkages/R/split.inputs.LINKAGES.R

```diff
@@ -17,10 +17,27 @@
 ##' @return files split up climate files
 ##' @export
 ##' 
-split.inputs.LINKAGES <- function(settings, start.time, stop.time) {
+split.inputs.LINKAGES <- function(settings, start.time, stop.time, inputs) {
   
-  new.met <- paste0(settings$rundir, "/climate.Rdata")  # doesn't do anything but write stuff to README
-  settings$run$inputs$met$path <- new.met  #HACK
+  #### Load all temp and precip met for site
+  met <- inputs$met$path
+  load(met)
   
-  return(settings$run$inputs)
+  #### Add years for indexing
+  temp.mat.years <- cbind(temp.mat,seq(850,2010,1))
+  precip.mat.years <- cbind(precip.mat,seq(850,2010,1))
```

Comment:
years should probably should go in met2model. This would break for any met other than PalEON 1